### PR TITLE
refactor(rxOptionTable): EO-1137 Refactor Option Table

### PR DESCRIPTION
**Pull request by sundip**
JIRA: https://jira.rax.io/browse/EO-1137

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- ~~[ ] QE LGTM~~ (No change to functional javascript)

Before
------------
before

After
------------
after

### CODE REVIEW

**Comment by CITguy** [comment]
Dev LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by sundip** [comment]
@ElementE when you get a moment, can you please take a look at this PR.  It's a small one.

File: src/elements/Forms/docs/forms.docs.html [hunk 1]
```diff
@@ -726,6 +726,10 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
 <h3><rx-permalink>Option Tables</rx-permalink></h3>
 
 <h4><rx-permalink>Radio Option Tables</rx-permalink></h4>
+<p>
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: This paragraph would fit better under the `<h3>` above. Please update.

**Comment by ElementE** [comment]
Design LGTM

**Comment by parlarjb** [comment]
@sundip this needs a rebase

**Pull request merged**